recipes/libpfm4/all/conanfile.py

```diff
@@ -0,0 +1,60 @@
+from conan import ConanFile, tools
+import os
+
+
+class Libpfm4Conan(ConanFile):
+    name = "libpfm4"
+    license = "MIT"
+    homepage = "https://github.com/wcohen/libpfm4"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = (
+        "a helper library to program the performance monitoring events")
+    topics = ("perf", "pmu", "benchmark", "microbenchmark")
+
+    # Binary configuration
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    # Sources are located in the same place as this recipe, copy them to the recipe
+    exports_sources = "src/*"
+    _source_subfolder = 'sources'  # Required to build at least 2.12.1
```

Comment:
```suggestion
    def layout(self):
        basic_layout(self, src_folder="src")
```
Nothing to be copied or exported if you are using new tools.
https://docs.conan.io/en/latest/reference/conanfile/tools/layout.html